README.md

```diff
@@ -4,7 +4,7 @@
 
 # Welcome to this repository!
 
-This repository stores the [__Epiverse TRACE Blueprints__](index.qmd) for software development. You can:
+This repository stores the [__Epiverse TRACE-Blueprints__](index.qmd) for software development. You can:
```

Comment:
Is this what was intended?

```suggestion
This repository stores the [__Epiverse-TRACE Blueprints__](index.qmd) for software development. You can:
```

I don't completely understand the meaning of the dash in the context otherwise